ajax/libs/nipplejs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "nipplejs",
+  "version": "0.6.7",
+  "filename": "nipplejs.min.js",
+  "description": "A virtual joystick for touch capable interfaces",
+  "author": "Yoann Moinet <yoann.moinet@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yoannmoinet/nipplejs.git"
+  },
+  "homepage": "http://yoannmoinet.github.io/nipplejs",
+  "keywords": [
+    "virtual",
+    "joystick",
+    "touch"
+  ],
+  "npmName": "nipplejs",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
+      ]
+    }
+  ]
```

Comment:
@ajay2507 

1. Yes, you need to put `close #11056, cc @yoannmoinet` in the commit description and not the title. You do it like this when making a commit:

```
$ git ci -m "Add nipplejs@0.6.7 w/ git auto-update" -m "close #11056, cc @yoannmoinet"
```

2. Not just the file rule, you need to replace this:

```
"npmName": "nipplejs",
  "npmFileMap": [
    {
      "basePath": "dist",
      "files": [
        "**/*"
      ]
    }
  ]
```
with this:
```
"autoupdate": {
    "source": "git",
    "target": "https://github.com/yoannmoinet/nipplejs.git",
    "fileMap": [
      {
        "basePath": "dist",
        "files": [
          "**/*.!(eslintrc)"
        ]
      }
    ]
  }
```